srcpkgs/pistol/template

```diff
@@ -0,0 +1,25 @@
+# Template file for 'pistol'
+pkgname=pistol
+version=0.5.2
+revision=1
+build_style=go
+go_import_path=github.com/doronbehar/pistol
+go_package="${go_import_path}/cmd/pistol"
+hostmakedepends="ruby-asciidoctor"
+makedepends="file-devel libmagic"
+depends=libmagic
```

Comment:
this `depends` is not required, xbps-src detects linked libraries and generates the correct minimum version dependencies for those.

See: https://github.com/void-linux/void-packages/actions/runs/16178189322/job/45863891884?pr=56215#step:7:400